### PR TITLE
bpo-36969: Make PDB args command display keyword only arguments

### PR DESCRIPTION
**Pull request by remilapeyre**
https://bugs.python.org/issue36969

### CODE REVIEW

File: Lib/pdb.py [hunk 1]
```diff
@@ -1132,9 +1132,9 @@ def do_args(self, arg):
         """
         co = self.curframe.f_code
         dict = self.curframe_locals
-        n = co.co_argcount
-        if co.co_flags & 4: n = n+1
-        if co.co_flags & 8: n = n+1
+        n = co.co_argcount + co.co_kwonlyargcount
```
Comment (pablogsal) [inline Lib/pdb.py:1135]: You also need to add co.co_posonlyargcount to handle PEP570 (check the issue). Although this will difficult backporting, so maybe you want this to be done in a separate PR.

Comment (remilapeyre) [reply Lib/pdb.py:1135]: Thank @pablogsal, it may be simple to do so in a separate PR I think.

File: Lib/pdb.py [hunk 2]
```diff
@@ -1132,9 +1132,9 @@ def do_args(self, arg):
         """
         co = self.curframe.f_code
         dict = self.curframe_locals
-        n = co.co_argcount
-        if co.co_flags & 4: n = n+1
```
Comment (pablogsal) [inline Lib/pdb.py:1136]: Thanks for removing these evil magic numbers!

**Review by pablogsal — APPROVED** [review]

File: Lib/pdb.py (hunk 2 above)
Comment (remilapeyre) [reply Lib/pdb.py:1136]: Definitely the improvement Python needed ^^
